src/routes/docs/products/network/+page.markdoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: article
+title: Network
+description: Discover Appwrite’s network architecture with global regions, edge nodes, and optimized routing. Explore how it ensures low latency, reliable performance, and scalable infrastructure for modern applications.
+---
+
+Appwrite's network is designed to deliver low-latency, high-performance experiences for developers and end-users alike. It leverages a robust Content Delivery Network (CDN) with edge locations across multiple regions to ensure fast and reliable data delivery.
+
+With distributed infrastructure and multiple deployment regions, Appwrite enables developers to build globally scalable applications while maintaining data sovereignty. Its architecture integrates seamlessly with APIs, storage, and databases, optimizing both speed and availability.
+
+[NETWORK MAP HERE - REGIONS, EDGES AND POPs INCLUDED]
+
+# Components {% #components %}
+The Appwrite network is composed of multiple components that work together to deliver a seamless experience for developers and end-users. These components include:
+
+{% cards %}
+{% cards_item href="/docs/products/network/regions" title="Regions" %}
+Multi-region deployments for geo-redundancy, compliance and low-latency.
+{% /cards_item %}
+{% cards_item href="/docs/products/network/edges" title="Edges" %}
+Edge nodes for fast request processing and reduced round-trip times.
+{% /cards_item %}
+{% cards_item href="/docs/products/network/cdn" title="CDN" %}
+Distributed caching with optimized routing for faster, more reliable data delivery.
+{% /cards_item %}
+{% cards_item href="/docs/products/network/endpoints" title="Entrypoints" %}
+Distributed caching with optimized routing for faster, more reliable data delivery.
+{% /cards_item %}
+{% /cards %}
+
+## Region vs Edge {% #region-vs-edge %}
```

Comment:
This might work better as a sub-section of Architecture

Reply:
We can once we have more content for a full architecture page